check for correct ready state/app.js

```diff
@@ -0,0 +1,15 @@
+const xhr = new XMLHttpRequest();
+
+xhr.onreadystatechange = function() {
+  if (xhr.readyState === 4 && xhr.status === 200) {
+   
+    console.log("Request was successful!");  
+    const sidebar = document.getElementById('sidebar');    
+    sidebar.innerHTML = xhr.responseText;
+  } else if (xhr.readyState === 4 && xhr.status !== 200) {
+    console.log("There was an error with the request: " + xhr.status);
+  }
+};
+
+xhr.open('GET', 'sidebar.html');
```

Comment:
I get error when running the files
<img width="687" alt="image" src="https://github.com/user-attachments/assets/086c9d47-930a-4348-9bb4-89d8ccd46459">

I think this should be
```suggestion
xhr.open('GET', 'index.html');
```